File: app/src/main/java/com/lemytho/app/ui/FunnyNames.kt
package com.lemytho.app.ui

/** Pseudos amusants (thème espion/enquête), partagés entre jeu local et multijoueur. */
object FunnyNames {
    val NAMES = listOf(
        "Sherlock", "Columbo", "Mata Hari", "Arsène", "Le Fouineur",
        "Hercule", "Miss Marple", "Le Corbeau", "Tête Brûlée", "L'Indic",
        "La Taupe", "Double Jeu", "Mr X", "La Silhouette", "L'Ombre",
        "Baron Noir", "Professeur", "La Belette", "Cervelle", "L'Espionne"
    )

    fun random(): String = NAMES.random()
}

File: web/src/funnyNames.ts
// Pseudos amusants (thème espion/enquête), miroir de FunnyNames.kt.

const NAMES = [
  "Sherlock",
  "Columbo",
  "Mata Hari",
  "Arsène",
  "Le Fouineur",
  "Hercule",
  "Miss Marple",
  "Le Corbeau",
  "Tête Brûlée",
  "L'Indic",
  "La Taupe",
  "Double Jeu",
  "Mr X",
  "La Silhouette",
  "L'Ombre",
  "Baron Noir",
  "Professeur",
  "La Belette",
  "Cervelle",
  "L'Espionne",
];

export function randomFunnyName(): string {
  return NAMES[Math.floor(Math.random() * NAMES.length)];
}
